### ISSUE TITLE
qiskit.aqua.components.multiclass_extensions has no newer equivalent

### ISSUE THREAD

**Issue by sri1905** [Qiskit/qiskit-machine-learning#256]
### What is the expected enhancement?

The migration guide at qiskit.aqua shows there is no equivalent for multiclass_extensions. It would be great if the new qiskit provides this feature too. Do let me know if something already exists. Thank you !

**Comment by adekusar-drl** [comment]
@sri1905 I believe there's no need to port `multiclass_extensions`. If you want to run multi-label classification via QSVC (is it what you are looking for?) then everything should work out of box. For example a 3-label classification on the `iris` dataset:
```python
from qiskit import Aer
from qiskit.circuit.library import ZZFeatureMap
from qiskit.utils import QuantumInstance
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split

from qiskit_machine_learning.algorithms import QSVC
from qiskit_machine_learning.kernels import QuantumKernel

if __name__ == '__main__':
    x, y = load_iris(return_X_y=True)
    samples = 30

    # 150 samples, 3 classes
    # iris dataset contains 50 samples of each class and samples are sorted by class label
    # so we sample a balanced sub-dataset of `samples` samples in total
    indices = [int(50 * (i % 3) + (float(i) / samples * 50)) for i in range(samples)]

    features = x[indices]
    labels = y[indices]

    train_features, test_features, train_labels, test_labels = train_test_split(features, labels, test_size=5, shuffle=False)

    qi = QuantumInstance(backend=Aer.get_backend('aer_simulator_statevector'),
                         seed_simulator=12345,
                         seed_transpiler=12345)

    qk = QuantumKernel(feature_map=ZZFeatureMap(feature_dimension=4), quantum_instance=qi)
    model = QSVC(quantum_kernel=qk)
    model.fit(train_features, train_labels)
    predicts = model.predict(test_features)
    print("true    :", test_labels)
    print("predicts:", predicts)
    score = model.score(test_features, test_labels)
    print("score   :", score)
```
Produces on my laptop:
```
true    : [1 2 0 1 2]
predicts: [0 2 0 1 2]
score   : 0.8
```
Please let me know if you have questions.

**Comment by sri1905** [comment]
@adekusar-drl thank you for your example. It does show how to perform classification with 3 classes. However, I believe there is an issue with four or more classes as @jonvet demonstrated in #251. I shall implement once and let you know if I face any errors/irregularities. Thank you

**Comment by adekusar-drl** [comment]
@sri1905 Was this issue resolved? I'm not closing #251meanwhile.

**Comment by sri1905** [comment]
> @sri1905 Was this issue resolved? I'm not closing #251meanwhile.

Yes, this issue can be closed. Thank you
